fossunited/chapters/doctype/chapter_event_member/chapter_event_member.py

```
@@ -0,0 +1,30 @@
+
+    if TYPE_CHECKING:
+        from frappe.types import DF
+
+        email: DF.Data | None
+        full_name: DF.Data | None
+        member: DF.Link | None
+        parent: DF.Data
+        parentfield: DF.Data
+        parenttype: DF.Data
+        role: DF.Literal[
+            "Volunteer",
```

Ruff (0.15.6)

[error] Undefined name `Volunteer`

(F821)